pyccel/codegen/wrapper/fortran_to_c_wrapper.py

```diff
@@ -483,6 +484,8 @@ def _wrap_Variable(self, expr):
                           original_function = expr)
             return expr.clone(expr.name, new_class = BindCArrayVariable, wrapper_function = func,
                                 original_variable = expr)
+        else:
+            raise NotImplementedError("Lists cannot be wrapped yet")
```

Comment:
What about sets?

Reply:
updated 296ca4e2ad0d07effc7402f327d7087e964b1224

Reply:
NDArray is a container that can be wrapped. Why not use the variable to clarify the error message:
```python
            raise NotImplementedError("Objects of type {expr.class_type} cannot be wrapped yet")
```

Reply:
updated c7d984cc6fc3ffbb26a030a3e7562eb212872497

Reply:
@jalalium Can we close this conversation?